pkg/migrations/op_set_fk.go

```diff
@@ -149,6 +149,13 @@ func (o *OpSetForeignKey) Rollback(ctx context.Context, conn *sql.DB) error {
 }
 
 func (o *OpSetForeignKey) Validate(ctx context.Context, s *schema.Schema) error {
+	name := TriggerName(o.Table, TemporaryName(o.Column))
+	if len(name) > MaxNameLength {
+		return InvalidNameLengthError{
+			Identity: name,
+			Max:      MaxNameLength,
+		}
+	}
```

Comment:
I don't think it makes sense to check the names of the triggers. The trigger functions are internal to `pgroll` and their names won't mean anything to the users starting the migration.

I would remove all the checks for trigger name length and just check the names of the user-defined parts of migrations like column and constraint names.